### ISSUE TITLE
Possibility to declare default color when export SVG icons

### ISSUE THREAD

**Issue by PixelT** [avocode/avocode#1499]
As in title - optionally possibility to define default svg icon(s) color when export

**Comment by vojtatranta** [comment]
I don't see quite benefit in this enhancement. You still can easily change color of SVG in CSS and that's what you should do.

**Comment by PixelT** [comment]
Yes, I would like to update this issue to: Possibility to remove fill, width & height attributes in SVG file (because all of this properties you can set in CSS).

**Comment by vojtatranta** [comment]
This makes sense.

Would it be OK if it was some global settings regardless of individual designs? Like global export option?

**Comment by PixelT** [comment]
I think yes, but maybe global settings for all designs in single project, instead of each of them (designs) individually.

+ add also stroke attribute to list above
